lib/server.ts

```diff
@@ -271,6 +265,81 @@ async function main(){
         },
     );
 
+    app.post(
+        '/config',
+        jsonParser,
+        async (
+            req: express.Request,
+            res: express.Response,
+            next: express.NextFunction,
+        ) => {
+            try {
+                // first tear down existing autokit configuration
+                await autoKit.teardown();
+
+                // each field that is specified, change it in file
+                for(let field in req.body){
+                    console.log(`Trying to change ${field} to ${req.body[field]}`)
+                    autokitConfig[field] = req.body[field];
```

Comment:
Nit: Maybe we can catch the error for values that are not valid keys of the autokit config?